README.md

```diff
@@ -65,7 +65,9 @@ status of running services, errors, logs, and more.
 **Follow along:** [@tilt_dev](https://twitter.com/tilt_dev) on Twitter. Updates
 and announcements on the [Tilt blog](https://blog.tilt.dev).
 
-**Help us make Tilt even better:** Run `tilt analytics opt in` to enable telemetry, so we can improve Tilt on every platform. Details in ["What does Tilt send?"](http://docs.tilt.dev/telemetry_faq.html).
+**Help us make Tilt even better:** Tilt sends anonymized usage data,
```

Comment:
This bolded section feels a little weird now that it's opt out. Maybe just renaming it to "opt out of analytics"?

Reply:
but...we don't want people to opt out of analytics? so it feels like a weird call to action

Reply:
Yes, but the result of _this_ call to action is that they don't help us make Tilt better because they'd be opting out of analytics.

It's not a blocking comment I just think this reads strangely. Not sure what the answer is though.

Reply:
ah, i see. i'll just remove this sentence.